doc/sphinxman/source/introduction.rst

```diff
@@ -472,114 +472,85 @@ For more details, see Tables :ref:`Energy <table:energy_gen>`,
+    | Mk-MRPT2                | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRCCSD               | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRCCSD(T)            | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | RAS-CI                  | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | CASSCF, RASSCF          | RHF/ROHF       | CONF/DF           | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | SAPT                    | RHF            | CONV              | ---          | ---            | threaded                    |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | ADC(2)                  | RHF            | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | EOM-CC2                 | RHF            | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | EOM-CCSD                | RHF/UHF/ROHF   | CONV              | RHF/UHF/ROHF | CONV           | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | EOM-CC3                 | RHF/UHF/ROHF   | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
```

Comment:
@lothian, are the gradient capabilities for eom-cc2, eom-ccsd, & eom-cc3 lines correct (none/rhf/uhf/rohf)?